src/main/java/com/epam/aidial/core/log/GfLogStore.java

```diff
@@ -96,11 +99,10 @@ private void append(ProxyContext context, LogEntry entry) {
             append(entry, "\"", false);
         }
 
-        String executionPath = context.getExecutionPath();
+        List<String> executionPath = context.getExecutionPath();
         if (executionPath != null) {
-            append(entry, ",\"execution_path\":\"", false);
-            append(entry, executionPath, true);
-            append(entry, "\"", false);
+            append(entry, ",\"execution_path\":", false);
+            append(entry, ProxyUtil.MAPPER.writeValueAsString(executionPath), true);
```

Comment:
do you want to see a json array in the log? append works only for a string

Reply:
["x", "y"] -> willl be encoded as [""x"", ""y""]

Reply:
fixed

Reply:
["x"y"] -> ["x"y"] - incorrect
["x"y"] -> ["x\"y"] - correct

You need to escape each element in a list.